lib/src/main/kotlin/com/lemonappdev/konsist/core/declaration/KoDeclarationImpl.kt

```diff
@@ -10,7 +10,10 @@ import org.jetbrains.kotlin.psi.psiUtil.isPublic
 import org.jetbrains.kotlin.psi.psiUtil.isTopLevelKtOrJavaMember
 import kotlin.reflect.KClass
 
-internal abstract class KoDeclarationImpl(private val ktTypeParameterListOwner: KtTypeParameterListOwner) :
+internal abstract class KoDeclarationImpl(
+    private val ktTypeParameterListOwner: KtTypeParameterListOwner,
+    open val parent: KoBaseDeclarationImpl?,
```

Comment:
The `parent` property in `KoDeclarationImpl constructor` should not be opened, just `val parent`